deployment/ansible/roles/nyc-trees.tiler/templates/upstart-nyc-trees-tiler.conf.j2

```diff
@@ -0,0 +1,11 @@
+description	"nyc-trees-tiler"
+
+start on ({{ "vagrant-mounted" if developing_or_testing else "local-filesystems and net-device-up IFACE!=lo" }})
+stop on shutdown
+
+respawn
+setuid nyc-trees
+chdir {{ tiler_home }}
+env HOME="/var/lib/nyc-trees"
+
+exec envdir /etc/nyc-trees.d/env node server.js > {{ tiler_log }}
```

Comment:
This should be changed to the following so that `STDERR` and `STDOUT` are directed:

```
exec envdir /etc/nyc-trees.d/env node server.js >> {{ tiler_log }} 2>&1
```

Also need the `>>` so that we append vs. truncate the logs.